src/buttons/radio.spec.ts

```diff
@@ -312,11 +316,35 @@ describe('ngbRadioGroup', () => {
     expect(getGroupElement(fixture.nativeElement)).toHaveCssClass('ng-valid');
     expect(getGroupElement(fixture.nativeElement)).not.toHaveCssClass('ng-invalid');
   });
+
+  it('should disable label and input when it is disabled using reactive forms', () => {
```

Comment:
Could you also add a test for template-driven forms? I _know_ that the current implementation should work, but better to be safe than sorry...